src/index.test.ts

```diff
@@ -4,43 +4,73 @@ import { Localize, StaticLocalize } from './index'
 describe('StaticLocalize', function () {
   describe('#constructor', function () {
     it('should throw an error if the locales option is not an object', function () {
-      Assert.fail('Missing test')
+      const locales = 'i am not an object'
+      const localeConstructorCall = () => { return new StaticLocalize({ locales }) }
+
+      Assert.throws(localeConstructorCall, Error, 'Langs is not an object')
     })
   })
 
   describe('#locale', function () {
     it('should throw if the locale being set is not supported', function () {
-      Assert.fail('Missing test')
+      const locales = { fr: { 'Hello!': 'Bonjour!' } }
+      const localize = new StaticLocalize({ locales })
+      const setLocale = () => localize.locale = 'pt'
+
+      Assert.throws(setLocale, Error, 'Locale unsupported [pt]')
     })
   })
 
   describe('#isSupportsLocale', function () {
```

Comment:
I accidentally misspelled this method name. Please rename to the correct name: `#isLocaleSupported`.